app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -38,11 +38,13 @@ export default class ChannelNavBar extends PureComponent {
         const padding = {paddingHorizontal: 0};
 
         let height;
+        let canHaveSubtitle = true;
         switch (Platform.OS) {
         case 'android':
             height = ANDROID_TOP_PORTRAIT;
-            if (DeviceTypes.IS_TABLET) {
+            if (!DeviceTypes.IS_TABLET && isLandscape) {
```

Comment:
I changed this here because in a previous change it was changed from `isLandScape` to `DeviceType.IS_TABLED`, which I think doesn't make sense, now, I only use the landscape size when is landscape and is not a tablet. And for landscape views I add the `canHaveSubtitle = false` that I pass to the ChanneTitle to show or not subtitle there.

Reply:
this is changing the current behavior where the height of the navbar is consistent on tablets.. I think you need to handle this in a different way

Reply:
Restoring the previous behavior